Spigot-API-Patches/0167-Add-more-Zombie-API.patch

```diff
@@ -0,0 +1,62 @@
+From 7ac07ac07ac07ac07ac07ac07ac07ac07ac07ac0 Mon Sep 17 00:00:00 2001
+From: BillyGalbreath <Blake.Galbreath@GMail.com>
+Date: Sun, 7 Oct 2018 04:29:51 -0500
+Subject: [PATCH] Add more Zombie API
+
+
+diff --git a/src/main/java/org/bukkit/entity/Zombie.java b/src/main/java/org/bukkit/entity/Zombie.java
+index 7ac07ac07ac0..7ac07ac07ac0 100644
+--- a/src/main/java/org/bukkit/entity/Zombie.java
++++ b/src/main/java/org/bukkit/entity/Zombie.java
+@@ -48,4 +48,48 @@ public interface Zombie extends Monster {
+      */
+     @Deprecated
+     public Villager.Profession getVillagerProfession();
++
++    // Paper start
++    boolean isDrowning();
```

Comment:
Can you add some Javadocs here? :)

Reply:
I always miss the first one because the `// Paper` comment makes my eyes overlook it :P